widget/class-widget.php

```diff
@@ -61,6 +64,12 @@ public function widget( $args, $instance ) {
             )
          );
 
+         // Set default title if empty
+         $instance['title'] = empty( $instance['title'] ) ? 'Flexible Volunteer Opportunities' : $instance['title'];
+
+         // Get URL of page that [flexible_volunteer_opps] shortcode was used
+         $allOppsPageLink = get_permalink( $wpdb->get_results( 'SELECT ID FROM ' . $wpdb->base_prefix . 'posts WHERE post_content = "[flexible_volunteer_opps]" AND post_parent = 0')[0]->ID );
```

Comment:
No need to check post parent now that i've added the check to ensure that the post is published. Before that check the query would return revisions of posts that contained a shortcode in previous versions.